packages/ui/resolution/resolution.js

```diff
@@ -183,13 +196,15 @@ function getScreenType (rez) {
  */
 function calculateFontSize (type) {
 	const scrObj = getScreenTypeObject(type);
+	const shouldScaleFontSize = config.findScreenTypeWithoutGoingOver ? workspaceBounds.width > scrObj.width && workspaceBounds.height > scrObj.height :
+		workspaceBounds.width < scrObj.width && workspaceBounds.height < scrObj.height;
 	let size;
 
 	if (orientation === 'portrait' && config.orientationHandling === 'scale') {
 		size = scrObj.height / scrObj.width * scrObj.pxPerRem;
 	} else {
 		size = scrObj.pxPerRem;
-		if (workspaceBounds.width > scrObj.width && workspaceBounds.height > scrObj.height ) {
+		if (orientation === 'landscape' && config.fontSizeHandling === 'scale' && shouldScaleFontSize) {
```

Comment:
Let's move`config.fontSizeHandling === 'scale'` condition to `shouldScaleFontSize` at line 199.

Reply:
It's fixed.